.github/workflows/deploy-dev.yml

```diff
@@ -1,10 +1,8 @@
 name: Build-Deploy
 on:
-  push:
-    branches-ignore:
-      - 'master'
-      - 'tmp/*'
   workflow_dispatch:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
```

Comment:
Dette er default, så holder å bare bruke `pull_request` hvis vi vil:
`By default, a workflow only runs when a pull_request_target's activity type is opened, synchronize, or reopened`